s3/src/main/java/ch/cyberduck/core/s3/S3PathContainerServiceFactory.java

```diff
@@ -0,0 +1,53 @@
+package ch.cyberduck.core.s3;/*
```

Comment:
Copyright formatter does this always wrong…

Reply:
done.